examples/kubernetes/1.10/cilium-aws-secret.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Do we really need this file? This can be created by the users with `kubectl create secret --from-file....`

Reply:
IMHO is easy to understand if we use the standard AWS way so users can figure it out easily what they need to complete.

Reply:
Can you explain this better? What is the standard AWS way? What about users that are not running AWS? Why would they have a AWS secrets in their cluster?

Reply:
Maybe it's the location of the file that is misleading? It is very unlikely that this will ever be k8s version specific so maybe we can add it to `addons/aws-security`?